code/modules/mob/living/simple_animal/abnormality/zayin/quietday.dm

```diff
@@ -0,0 +1,179 @@
+	"The young traveler gasped for air, coughing and sputtering, and demanded to know why the old man had done such a thing. The old man replied, 'When you were underwater, what did you want more than anything else in the world?'.",
+	"The young traveler thought for a moment and then replied, 'Air. I wanted air more than anything.'",
+	"The old man smiled and said, 'Exactly. The secret to a happy and fulfilled life is to want something as much as you wanted air when you were underwater. That kind of drive and determination will help you achieve anything you set your mind to.'")
+
+	var/list/wife = list("Certainly, I would be happy to share a story about my dear wife",
+	"My wife and I were both from L Corporation, and we worked together in the same office for many years. We had a special bond, not just as husband and wife, but also as coworkers who shared a passion for our profession.",
+	"One day, she looked extremely stressed.",
+	"I asked her if everything was okay, and she confided in me that she was worried about a mistake she had made in one of the reports.",
+	"I am a very calm man, I reassured her that we would work together to fix the mistake and make sure that everything was correct. We spent the rest of the night poring over the reports, double-checking to make sure that everything was accurate.",
+	"By the time we finished, it was already well past midnight. We were both exhausted, but also relieved and proud of the work we had done. We hugged each other and I told my wife how much I appreciated her dedication and hard work.",
+	"She was quite the fiery woman, but I loved her so very much. I still miss her this very day, I hope she made it out of the nest.")
+
+	var/list/dementia = list("I'm sorry, what was your name again? My memory isn't what it used to be.",
+	"I remember a time when we used to... oh, wait, where was I going with this?",
+	"Sometimes, I feel like my memory is slipping away. But I still have all these stories and experiences inside me that I want to share.",
+	"I'm having trouble finding the right words to express myself. Please bear with me.",
+	"I'm sorry, my mind isn't as sharp as it used to be. Could you please repeat what you said?",
+	"Oh dear, where did I put my glasses? I can't seem to find them anywhere.",
+	"Now, what was I going to say? It's right on the tip of my tongue...",
+	"I think I've seen you somewhere before, haven't I? Or am I mistaken?",
+	"Sometimes my mind feels like a jumbled mess. I wish I could straighten it all out."
+	)
+
+/mob/living/simple_animal/hostile/abnormality/quiet_day/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(pe == 0)
+		return
+	buff_given = work_type
+	TalkStart(user)
+
+/mob/living/simple_animal/hostile/abnormality/quiet_day/proc/TalkStart(mob/living/carbon/human/user)
+	switch(buff_given)
+		if(ABNORMALITY_WORK_INSTINCT)
+			for(var/line in war_story)
+				say(line)
+				SLEEP_CHECK_DEATH(50)
+				if(PlayerCheck(user) == TRUE)
```

Comment:
Make PlayerCheck return False outside of the `if` statements and change these to just `if(PlayerCheck(user))` please